addons/info/src/components/PropTable.js

```diff
@@ -1,10 +1,16 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import { Table, Td, Th } from '@storybook/components';
 import PropVal from './PropVal';
 import PrettyPropType from './types/PrettyPropType';
 
+const Table = props => <table style={{}} {...props} />;
```

Comment:
what's the point of adding an empty `style` object?

Reply:
right nothing, doesn't output anything either though

Reply:
Why not just use `table` then?

Reply:
I don't feel strongly about this, but I like the look of jsx when it's just components, vs components & html mixed.

I don't think there are any pros or cons to speak of really.

Reply:
One con is that it makes code less understandable at a first glance. If you really want camelcase, you can do this:

```
const Table = 'table'
```